hal/transport/RFM69/driver/new/RFM69_new.cpp

```diff
@@ -384,7 +443,16 @@ LOCAL bool RFM69_sendFrame(rfm69_packet_t *packet, const bool increaseSequenceCo
 	while (!RFM69_channelFree() &&
 	        ((hwMillis() - CSMA_START_MS) < MY_RFM69_CSMA_TIMEOUT_MS)) {
 		doYield();
+		
+		//A little delay before the next attempt
+		delay(1);
```

Comment:
Is 1ms the right amount of time to delay? Could we document why 1ms was chosen?

Reply:
I saw some CSMA realisations and all of them contains such delay. There is no reason check the air immediately after it is busy. Moreover it is nice to implement randomized delay... 1ms works fine in my tests but I do not insist on exacly this value.

Reply:
Sounds reasonable. Let's stick with the current value unless anyone else has an opinion.